operator/v1/types_kubeapiserver.go

```diff
@@ -33,6 +33,23 @@ type KubeAPIServerSpec struct {
 
 type KubeAPIServerStatus struct {
 	StaticPodOperatorStatus `json:",inline"`
+
+	// serviceAccountIssuers tracks history of used service account issuers.
+	// The item without expiration time represents the currently used service account issuer.
+	// The other items represents service account issuers that were used previously and are still being trusted.
```

Comment:
How long will the entry be trusted for once it is no longer the present entry?

Where does the configuration for this come from?

Reply:
The default is 24h, and it is hardcoded in the controller. I don't think we want to expose this to users to configure the expiration time.